funnel/models/project.py

```diff
@@ -191,6 +192,13 @@ class Project(UuidMixin, BaseScopedNameMixin, Model):
         read={'all'},
         datasets={'primary', 'without_parent', 'related'},
     )
+
+    thumbnail_image = with_roles(
```

Comment:
I think we can drop the `with_roles` wrapper entirely. There's no need for any template to access the contents of this field.

We also don't want this to be loaded most of the time, so the column definition needs to be `sa.orm.mapped_column(sa.LargeBinary, nullable=True, deferred=True)`